code/game/machinery/doors/airlock.dm

```diff
@@ -745,8 +748,7 @@
 /obj/machinery/door/airlock/attack_paw(mob/user, list/modifiers)
 	return attack_hand(user, modifiers)
 
-/obj/machinery/door/airlock/proc/on_attack_hand(mob/user, list/modifiers)
-	SIGNAL_HANDLER
+/obj/machinery/door/airlock/proc/on_attack_hand(atom/source, mob/user, list/modifiers)
 	return attack_hand(user, modifiers)
```

Comment:
One last step, bring back `SIGNAL_HANDLER`--it's still important, as these are still hooked to RegisterSignal.

Reply:
it's got a bunch of errors with SIGNAL_HANDLER

Reply:
Replace with:

```dm
SIGNAL_HANDLER
INVOKE_ASYNC(src, .proc/attack_hand, user, modifiers)
```